samples/Cryptocash.Api/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Infrastructure.Persistence/CryptocashDbContext.g.cs

```diff
@@ -116,6 +117,18 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             if (type != null)
             {
                 ((INoxDatabaseConfigurator)_dbProvider).ConfigureEntity(codeGeneratorState, new EntityBuilderAdapter(modelBuilder.Entity(type)), entity);
+
+                if (entity.Keys.Count == 1 &&
+                    entity.GetAttributesToLocalize().Any())
+                {
+                    type = codeGeneratorState.GetEntityType(entity.LocalizedName);
```

Comment:
sorry you will need to update this, because we have refactor code generator to be `NoxCodeGenConventions` and not depending on the assembly, you will have to use something like this:
`_clientAssemblyProvider.GetType(_codeGeneratorState.GetEntityTypeFullName(entity.LocalizedName));`

Reply:
Done